### DISCUSSION TITLE
qt msvc static template

### DISCUSSION THREAD

**Discussion by gmh5225** [build-cpp/cmkr#77] [Show and tell]
I used `cmkr`to create a qt msvc static template that may be useful to some people.
Thanks `cmkr` :)

QT5:
https://github.com/gmh5225/cmkr-qt5-msvc-static-template

QT6:
https://github.com/gmh5225/cmkr-qt6-msvc-static-template

**Comment by mrexodia** [comment]
That’s great, thanks! Just be mindful that Qt itself is usually compiled dynamically so it doesn’t make much sense to do this.
